src/index.html

```diff
@@ -0,0 +1,39 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css"
+        integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <title>Tv Shows</title>
+</head>
+
+<body>
+
+    <header class="d-flex justify-content-center">
+        <nav>
+            <ul class="nav-list d-flex">
+                <li class="link">TV Shows</li>
+                <li class="link">Categories</li>
+                <li class="link">Schedules</li>
+            </ul>
```

Comment:
- [ ] As pointed out by the previous reviewer, kindly include a logo in the site header so that It matches the original design